service/src/main/java/skills/skillLoading/SkillsLoader.groovy

```diff
@@ -974,6 +974,17 @@ class SkillsLoader {
                         numSkillsRequired: skillDef.numSkillsRequired,
                         totalPoints: skillDef.totalPoints,
                 )
+
+                Boolean groupDescriptionsOn = settingsService.getProjectSetting(skillDef.projectId, 'group-descriptions')?.value?.toBoolean()
+
+                if(groupDescriptionsOn) {
```

Comment:
I think would be nice to add a service tests to validate this logic, perhaps in ``src/test/java/skills/intTests/clientDisplay/SkillsDescriptionSpec.groovy``?